### ISSUE TITLE
swipe event blocked

### ISSUE THREAD

**Issue by gtouzeau** [watusi/jquery-mobile-iscrollview#159]
Hello,
I want to use jquery swipe event on iscrollview blocks but they are never catched.

Like if I use this simple code : 
`$('body').on("swiperight", function(event, ui) {
            alert('swipeRight');
        });` 
=> The alert message never show up.

Is there any option parameters or solutions to do it? 

Thank you.

**Comment by frankie-loves-jesus** [comment]
I don't think you should be using iscrollview anymore. See https://github.com/watusi/jquery-mobile-iscrollview/issues/118.
